### PR TITLE
Remove `Server` object

### PR DESCRIPTION
**Pull request by cpuguy83**
This had some weird shared responsibility with the PodController.
Instead just move the functionality to the PodController.

### CODE REVIEW

**Comment by cpuguy83** [comment]
ping @sargun

**Comment by cpuguy83** [comment]
Note that this no longer has the `InSync` call since the caller can wait on the cache to be in sync.

**Comment by cpuguy83** [comment]
I was considering adding another object (but as a separate PR)... probably called `Agent`... which basically manages the `PodController` and `Node`... so basically a bunch of the work that the CLI currently does.

File: vkubelet/podcontroller.go [hunk 1]
```diff
@@ -22,21 +22,22 @@ import (
 	"sync"
 	"time"
 
+	"github.com/cpuguy83/strongerrors"
+
```
Comment (sargun) [inline vkubelet/podcontroller.go]: Why the newline?

Comment (cpuguy83) [reply vkubelet/podcontroller.go]: Took did this for whatever reason

Comment (cpuguy83) [reply vkubelet/podcontroller.go]: *Tool

Comment (cpuguy83) [reply vkubelet/podcontroller.go]: *Tool

**Comment by cpuguy83** [comment]
Ok, now this actually works.

**Pull request merged**